submission-testing/src/submissionTester.ts

```diff
@@ -33,13 +33,13 @@ runnerImageContexts.forEach(async (context: string, language: string) => {
  * Begins submission evaluation and returns the ID of the submission.
  */
 export async function evaluateSubmission(code: string, language: string, challengeId: number, challengeName: string, userId: number, userName: string): Promise<string> {
-	// FIXME get the testcases from the challenges service.
-	const testCases: TestCase[] = [
-		// Input arrays are nested because it includes all parameters passed to the function (the first and only of which is the array to be sorted).
-		{"input": [[3, 6, 1]], "expectedOutput": [1, 3, 6]},
-		{"input": [[7, 5]], "expectedOutput": [5, 7]},
-		{"input": [[10, 20, 30]], "expectedOutput": [10, 20, 30]}
-	];
+	const parameters: any = JSON.parse(await request("http://challenges:3000/getChallengeParameters", {
+		method: "GET",
+		qs: {
+			challengeId: challengeId
+		}
+	}));
+	const testCases: TestCase[] = parameters.testCases;
```

Comment:
I tried testing your PR and it threw a JSON parsing error. It seems like the "testCases" that you are getting from the Challenges endpoint is structured differently compared to the hardcoded testCases you had before.

Reply:
Aside from that, it's all good to go!

Reply:
Resolved by https://github.com/jitli98/CS497Project/pull/16/commits/7c8d3b90344cf3af05e0a91dd73ad69be033ac21